#35 - main page management

kamilpiech97 opened the pull request:
This should close #35.
Added CRUD for main page sections like about me and counters. Additionally added possibility of managing visibility of main page sections: about me, counters, contact and banner.
Added some tests.

Preview:
![image](https://github.com/blumilksoftware/keating/assets/29115024/fc859fb1-b5c3-4dd7-8dad-1fbf40b62077)

EwelinaSkrzypacz reviewed (commented):
1. Validation fields like "title" and "value" is not translated.
2. In validation message about "typ sekcji" is text that "typ zajęć jest wymagane" - inconsistency in naming

![image](https://github.com/blumilksoftware/keating/assets/56546832/95418ea5-b9b4-4b20-82bb-06efea7af750)

3. When validation error occurs (like no title) but I fill other field like section type and description, after clicking "Utwórz" all field are cleared.

https://github.com/blumilksoftware/keating/assets/56546832/cb476e34-3755-4b20-ad6e-3e8a67955996

4. When I add section type "O mnie" and use WYSIWYG editor to add unordered list on the main page it's just text.

https://github.com/blumilksoftware/keating/assets/56546832/fba60f60-505a-401c-b2f6-1fe61717087a

kamilpiech97 commented:
> 1. Validation fields like "title" and "value" is not translated.
> 2. In validation message about "typ sekcji" is text that "typ zajęć jest wymagane" - inconsistency in naming
> 
> ![image](https://user-images.githubusercontent.com/56546832/281674635-95418ea5-b9b4-4b20-82bb-06efea7af750.png)
> 
> 3. When validation error occurs (like no title) but I fill other field like section type and description, after clicking "Utwórz" all field are cleared.
> 
>  simplescreenrecorder-2023-11-09_07.18.12.mp4 
> 4. When I add section type "O mnie" and use WYSIWYG editor to add unordered list on the main page it's just text.
> 
>  simplescreenrecorder-2023-11-09_10.10.22.mp4

1. translated
2. fixed
3. fixed
4. fixed - added typography plugin for tailwind
